### PR TITLE
fix(docker): use debian as base image instead of ubuntu

### PR DESCRIPTION
**Pull request by petrjasek**
there is some issue on ubuntu when doing npm install,
seems to work fine on debian

### CODE REVIEW

**Comment by petrjasek** [comment]
ping @actionless

File: Dockerfile [hunk 1]
```diff
@@ -16,12 +16,6 @@ curl libfontconfig nodejs npm nginx \
 
 RUN npm -g install grunt-cli bower
 
-# Set the locale
-RUN locale-gen en_US.UTF-8
-ENV LANG en_US.UTF-8
-ENV LANGUAGE en_US:en
-ENV LC_ALL en_US.UTF-8
```
Comment (actionless) [inline Dockerfile]: that locale is set in the parent image?

File: Dockerfile [hunk 2]
```diff
@@ -1,5 +1,5 @@
 # import base image
-FROM ubuntu:trusty
+FROM buildpack-deps:jessie-curl
```
Comment (actionless) [inline Dockerfile]: is there a way to use some more "official" debian image?

File: Dockerfile (hunk 1 above)
Comment (petrjasek) [inline Dockerfile]: there is no locale-gen in debian

File: Dockerfile (hunk 2 above)
Comment (petrjasek) [inline Dockerfile]: define more official, this one is from docker-library

**Comment by actionless** [comment]
@ioanpocol , seems like with you latest change it's working fine now, can we close this then?

File: Dockerfile (hunk 2 above)
Comment (actionless) [inline Dockerfile]: https://hub.docker.com/_/debian/

File: Dockerfile (hunk 1 above)
Comment (actionless) [inline Dockerfile]: can you paste here `locale` output from there then?

(we still could proceed with migrating to debian in order to shrink resulting image size)

**Comment by petrjasek** [comment]
not sure that fix is working, check master

File: Dockerfile (hunk 1 above)
Comment (petrjasek) [inline Dockerfile]: no worries, I will put it back, my bad thinking it's not in debian

**Comment by actionless** [comment]
@petrjasek 
```
/var/opt/sd_docker/master# ./docker-compose.sh logs superdesk
Attaching to deploymaster_superdesk_1
superdesk_1 | All services are started :3
superdesk_1 | 11:13:58 system | rest.1 started (pid=44)
superdesk_1 | 11:13:58 system | work.1 started (pid=47)
superdesk_1 | 11:13:58 system | beat.1 started (pid=48)
superdesk_1 | 11:13:58 system | wamp.1 started (pid=46)
superdesk_1 | 11:13:58 rest.1 | [2016-04-06 11:13:58 +0000] [45] [INFO] Starting gunicorn 19.4.5
superdesk_1 | 11:13:58 rest.1 | [2016-04-06 11:13:58 +0000] [45] [INFO] Listening at: http://0.0.0.0:5000 (45)
superdesk_1 | 11:13:58 rest.1 | [2016-04-06 11:13:58 +0000] [45] [INFO] Using worker: sync
superdesk_1 | 11:13:58 rest.1 | [2016-04-06 11:13:58 +0000] [60] [INFO] Booting worker with pid: 60
superdesk_1 | 11:13:58 rest.1 | [2016-04-06 11:13:58 +0000] [62] [INFO] Booting worker with pid: 62
superdesk_1 | 11:13:58 wamp.1 | Failed to start the WebSocket server.
superdesk_1 | 11:13:58 wamp.1 | Traceback (most recent call last):
superdesk_1 | 11:13:58 wamp.1 |   File "/usr/local/lib/python3.4/dist-packages/superdesk/ws.py", line 27, in
create_server
superdesk_1 | 11:13:58 wamp.1 |     broker_url = config['BROKER_URL']
superdesk_1 | 11:13:58 wamp.1 | KeyError: 'BROKER_URL'
superdesk_1 | 11:13:58 wamp.1 | Failed to start the WebSocket server.
superdesk_1 | 11:13:58 wamp.1 | Traceback (most recent call last):
superdesk_1 | 11:13:58 wamp.1 |   File "/usr/local/lib/python3.4/dist-packages/superdesk/ws.py", line 27, in
create_server
superdesk_1 | 11:13:58 wamp.1 |     broker_url = config['BROKER_URL']
superdesk_1 | 11:13:58 wamp.1 | KeyError: 'BROKER_URL'
superdesk_1 | 11:13:58 system | wamp.1 stopped (rc=0)
superdesk_1 | 11:13:58 system | sending SIGTERM to rest.1 (pid 44)
superdesk_1 | 11:13:58 system | sending SIGTERM to beat.1 (pid 48)
superdesk_1 | 11:13:58 system | sending SIGTERM to work.1 (pid 47)
superdesk_1 | 11:13:58 rest.1 | [2016-04-06 11:13:58 +0000] [45] [INFO] Handling signal: term
superdesk_1 | 11:13:58 system | beat.1 stopped (rc=-15)
superdesk_1 | 11:13:58 system | work.1 stopped (rc=-15)
superdesk_1 | 11:13:59 rest.1 | [2016-04-06 11:13:59 +0000] [62] [INFO] Worker exiting (pid: 62)
superdesk_1 | 11:13:59 rest.1 | [2016-04-06 11:13:59 +0000] [60] [INFO] Worker exiting (pid: 60)
superdesk_1 | 11:13:59 rest.1 | [2016-04-06 11:13:59 +0000] [45] [INFO] Shutting down: Master
superdesk_1 | 11:13:59 system | rest.1 stopped (rc=-15)
```

**Comment by petrjasek** [comment]
merged a fix for that error https://github.com/superdesk/superdesk/pull/1906

**Comment by petrjasek** [comment]
seems like everything is working now, so no need for this

**Pull request closed without merging**